index.html

```diff
@@ -299,144 +316,67 @@ <h3 id="advertising-payload">Advertising payload</h3>
                   >After converting it to text</a
                 >
                 , it corresponds to "DIY-sensor". The name can be used to
-                identify your sensor.
+                identify your sensor. If you decide not to broadcast a name,
+                a default name "BTHome sensor" will be used. All names will
+                automatically get a postfix with the last 4 characters of the
+                MAC address to be able to identify different devices.
```

Comment:
This is a consumer detail and there is no need to add this to the spec. If we want, we should change the text to be explicit that consumers need to implement it like that.

```suggestion
                identify your sensor.
```

Reply:
agreed